betterlockscreen

```diff
@@ -161,7 +164,7 @@ postlock() {
 lockselect() {
 
     echof act "Running prelock..."
-    prelock &
+    prelock 
```

Comment:
We want the `&`. Even though we want the stuff in `prelock()` to execute, errors there can keep `lock()` from running. See [multilockscreen/#40](https://github.com/jeffmhubbard/multilockscreen/issues/40).

Reply:
this will be reverted.

Reply:
solved.